.circleci/config.yml

```diff
@@ -0,0 +1,213 @@
+version: 2.1
+
+aliases:
+  - &working_directory ~/merchant-center-application-kit
+
+  - &restore_yarn_cache
+    name: 'Restoring yarn cache'
+    keys:
+      - v1-yarn-cache-{{ checksum "yarn.lock" }}
+      - v1-yarn-cache
+
+  - &save_yarn_cache
+    name: 'Saving yarn cache'
+    key: v1-yarn-cache-{{ checksum "yarn.lock" }}
+    paths:
+      - ~/.cache/yarn
+
+executors:
+  node_10:
+    docker:
+      - image: circleci/node:10.15
+    working_directory: *working_directory
+  cypress:
+    working_directory: *working_directory
+    docker:
+      - image: 'cypress/base:10'
+        environment:
+          TERM: xterm
+
+commands:
+  install_dependencies:
+    description: 'Installing dependencies'
+    steps:
+      - checkout
+      - restore_cache: *restore_yarn_cache
+      - run: yarn install --pure-lockfile
+      - save_cache: *save_yarn_cache
+  build_artifacts:
+    description: 'Building artifacts'
+    steps:
+      - run: yarn build
+  save_workspace:
+    steps:
+      - persist_to_workspace:
+          root: *working_directory
+          paths:
+            - packages/**/dist
```

Comment:
Hmm don't we also need the `package.json` etc?

Reply:
We could yeah. But we checkout before restoring the workspace. I find that more intuitive. You checkout the code and restore what you intend to carry on. That means that you don't persist everything (inlcuding all the clutter).

Reply:
Ah right, I missed that part. Thanks